public/javascripts/nfl-data.js

```diff
@@ -90,17 +90,27 @@ function NFLData() {
   }
 
   function callKimonoAPI(position) {
-    $.ajax({
-      "url": '/nflData?position=' + position,
-      "dataType": "json"
-    })
-    .done(function (data) {
-      if (position == 'DEF') {
-        getTeams(data);
-      } else {
-        getPlayers(data);
-      }
-    });
+    if (NFLData.USE_CACHED_PLAYER_DATA) {
```

Comment:
Setting a flag to determine whether to use the cached json files or not. I didn't want to have to keep adding this back in.